src/data/list/basic.lean

```diff
@@ -3884,5 +3937,46 @@ theorem sum_map_mul_right {α : Type*} [semiring α] {β : Type*} (L : list β)
   (f : β → α) (r : α) :
   (L.map (λ b, f b * r)).sum = (L.map f).sum * r :=
 sum_map_hom L f $ add_monoid_hom.mul_right r
+.
+universes u v
+.
```

Comment:
```suggestion

universes u v

```
These appeared again; I guess you may as well move `nth_injective` into this PR as well so #3967 won't touch so many files.